### PR TITLE
add: criterion-related RUSTSEC ignore

### PR DESCRIPTION
**Pull request by Velfi**
*Description of changes:*
We're failing a security audit due to using Criterion in a benchmarking test. This ignores it for the purposes of the audit

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .github/workflows/audit.yaml [hunk 1]
```diff
@@ -18,13 +18,18 @@ jobs:
...
+          # (so we never invoke the problematic behavior that results from attempting to determine
+          # the local time of the current platform).
+          #
+          # Chrono's only use is as an optional feature in `aws-smithy-types-convert`, for
+          # the narrow use-case of converting from the SDK's `DateTime` into chrono types,
+          # and we never invoke the problematic behavior that results from attempting to
+          # determine the local time of the current platform.
+          --ignore RUSTSEC-2020-0071 \
+          --ignore RUSTSEC-2020-0159 \
+          # Criterion is only used to run benchmarks during CI
```
Comment (jdisanti) [inline .github/workflows/audit.yaml:34]: We should add some more detail to this to indicate serde_cbor is coming in through criterion, and perhaps a TODO comment and a tracking issue to update criterion and remove the ignore once criterion moves off of serde_cbor.

Comment (Velfi) [reply .github/workflows/audit.yaml]: I have updated it to link to the issue I created https://github.com/awslabs/smithy-rs/issues/1044

File: .handwritten [hunk 2]
```diff
@@ -23,5 +23,4 @@
 /CODEOWNERS
 /CONTRIBUTING.md
 /LICENSE
-/NOTICE
```
Comment (rcoh) [inline .handwritten:26]: did you mean to remove this?

**Review by rcoh — APPROVED** [review]

File: .handwritten (hunk 2 above)
Comment (Velfi) [reply .handwritten:26]: dangit

**Pull request merged**